pkg/ami/auto_resolver.go

```diff
@@ -42,7 +45,9 @@ func OwnerAccountID(imageFamily, region string) (string, error) {
 	switch imageFamily {
 	case api.NodeImageFamilyUbuntu1804:
 		return ownerIDUbuntu1804Family, nil
-	case api.NodeImageFamilyWindowsServer2019CoreContainer, api.NodeImageFamilyWindowsServer2019FullContainer:
+	case api.NodeImageFamilyWindowsServer2019CoreContainer,
+		api.NodeImageFamilyWindowsServer2019FullContainer,
+		api.NodeImageFamilyWindowsServer1909CoreContainer:
```

Comment:
Since this `case` clause has grown a bit, I think we should now use `isWindowsImage`.

Reply:
fixed, `IsWindowsImage` now in `default` case